charts/portworx/templates/portworx-csi.yaml

```diff
@@ -1,5 +1,7 @@
 {{- if (and (.Values.csi) (eq .Values.csi true))}}
 {{- $customRegistryURL := .Values.customRegistryURL | default "none" }}
+{{- $varImages := .Values.images | default dict }}
```

Comment:
Err.. no, the `dict` is Python's hasharray/map (e.g. equivalent of `var varImages := map[string]string`).
We'll associate it here as a _default value_, to define the _type_ of the `values/images` when this is missing from `values.yaml` file.  This is required as helm has no other way to determine the type of this variable.

Note that this is a common workaround for helm-charts, to prevent errors when dealing with complex structures in `values.yaml` file.